sample_apps/simple_app/run_example.md

```diff
@@ -0,0 +1,40 @@
+# run_example.sh - Setup and run test for simple_app program
+
+This document is a user's guide to the run_example.sh script that you can use
```

Comment:
If we like this new form of `run_example.sh`, this md-file will  be the quick-start how-to user's guide.